common/mock-radio.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Mock radio for interfacing with mock devices
+ */
+
+export interface MockUDPListener {
+  onUDPMessage(msg: Buffer, rinfo: RemoteAddressInfo): void;
+}
+
+export class RemoteAddressInfo {
+  address: string;
+  family: string;
+  port: number;
+  size: number;
+  constructor(address: string, family: string, port: number, size: number) {
+    this.address = address;
+    this.family = family;
+    this.port = port;
+    this.size = size;
+  }
+}
+
+// TODO(cjdaly) Builders
+export class DiscoveryData {
```

Comment:
Note those are discovery data that are specific to the sample, they shouldn't be part of the tool.

Reply:
Done